scriptmodules/ports/vvvvvv.sh

```diff
@@ -43,7 +43,22 @@ function install_vvvvvv() {
 }
 
 function configure_vvvvvv() {
+    local script
+    setConfigRoot "ports"
```

Comment:
This isn't needed - this is for ports that do not live in the scriptmodule/ports folder (eg libretro cores like setConfigRoot "ports")